luigi/static/visualiser/index.lte.html

```diff
@@ -0,0 +1,481 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Luigi Task Visualiser</title>
+        <script src="lib/jquery-1.10.0.min.js"></script>
+        <link href="lib/bootstrap3/css/bootstrap.min.css" rel="stylesheet">
+        <link href="lib/bootstrap3/css/bootstrap-theme.min.css" rel="stylesheet">
+        <script src="lib/bootstrap3/js/bootstrap.min.js"></script>
+        <link rel="stylesheet" href="css/tipsy.css">
+
+        <link href="css/AdminLTE.min.css" rel="stylesheet"/>
+        <link href="css/font-awesome.min.css" rel="stylesheet"/>
+        <link href="css/skin-green.min.css" rel="stylesheet"/>
+
+        <link href="lib/datatables/css/jquery.dataTables.min.css" rel="stylesheet"/>
+
+        <script src="http://d3js.org/d3.v3.js" charset="utf-8"></script>
+        <script src="js/dagre-d3.js"></script>
+        <script src="lib/mustache.js"></script>
+        <script src="js/luigi.d3.js"></script>
+        <script src="js/graph.js"></script>
+        <script src="js/visualiserApp.lte.js"></script>
+        <script src="js/tipsy.js"></script>
+
+        <script src="js/AdminLTE-app.min.js"></script>
+        <script src="lib/datatables/js/jquery.dataTables.min.js"></script>
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <style type="text/css">
```

Comment:
I just copied the inline style from index.d3.html.  I can definitely factor them into a separate file.  It would be good to do a more general refactor of the 3 themes at the same time as they are rather mixed up at the moment.